### PR TITLE
[AppServices]: Fix #16325 and #14624 : Expose HealthCheckPath SiteConfig in `Set-AzWebApp`, `Set-AzWebAppSlot`, `Get-AzWebApp` and `Get-AzWebAppSlot`

### LINKED ISSUE #16325: Set-AzWebApp: Expose HealthCheckPath SiteConfig
## Description of the new feature
The current Set-AzWebApp does not expose the SiteConfig "HealthCheckPath" parameter. We want to automate setting the "HealthCheckPath".

Note: there is a bug open because the "HealthCheckPath" remains null in the SiteConfig despite having a value:
https://github.com/Azure/azure-powershell/issues/14624

## Proposed implementation details (optional)

### PR DESCRIPTION
**Pull request by Kotasudhakarreddy**
#16325 , #14624 
## Description
The current Set-AzWebApp does not expose the SiteConfig "HealthCheckPath" parameter. We want to automate setting the "HealthCheckPath".

## Checklist

- [ ] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [ ] The title of the PR is clear and informative
- [ ] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [ ] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [ ] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
        - {Please put the link here}
    - [ ] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

**Review by BethanyZhou — APPROVED** [review]
LGTM

**Review by BethanyZhou — APPROVED** [review]

File: src/Websites/Websites/help/Set-AzWebApp.md [hunk 1]
```diff
@@ -24,8 +24,8 @@ Set-AzWebApp [[-AppServicePlan] <String>] [[-DefaultDocuments] <String[]>] [[-Ne
  [-ContainerRegistryUser <String>] [-ContainerRegistryPassword <SecureString>]
  [-EnableContainerContinuousDeployment <Boolean>] [-HostNames <String[]>] [-NumberOfWorkers <Int32>] [-AsJob]
  [-AssignIdentity <Boolean>] [-HttpsOnly <Boolean>] [-AzureStoragePath <WebAppAzureStoragePath[]>]
- [-AlwaysOn <Boolean>] [-MinTlsVersion <String>] [-FtpsState <String>] [-ResourceGroupName] <String>
- [-Name] <String> [-DefaultProfile <IAzureContextContainer>] [<CommonParameters>]
+ [-AlwaysOn <Boolean>] [-MinTlsVersion <String>] [-FtpsState <String>] [-HealthCheckPath <String>]
```
Comment (panchagnula) [inline src/Websites/Websites/help/Set-AzWebApp.md:27]: Don't expose this a parameter directly in the Set-AzWebapp. Please expose this in the  PSSite object this is identified & returned in the response & can be set using the -Webapp property that take a PSSite as input.

File: src/Websites/Websites/help/Set-AzWebAppSlot.md [hunk 2]
```diff
@@ -24,8 +24,8 @@ Set-AzWebAppSlot [[-AppServicePlan] <String>] [[-DefaultDocuments] <String[]>]
  [-AutoSwapSlotName <String>] [-NumberOfWorkers <Int32>] [-ContainerImageName <String>]
  [-ContainerRegistryUrl <String>] [-ContainerRegistryUser <String>] [-ContainerRegistryPassword <SecureString>]
  [-EnableContainerContinuousDeployment <Boolean>] [-AsJob] [-AssignIdentity <Boolean>] [-HttpsOnly <Boolean>]
- [-AzureStoragePath <WebAppAzureStoragePath[]>] [-AlwaysOn <Boolean>] [-MinTlsVersion <String>]
- [-FtpsState <String>] [-ResourceGroupName] <String> [-Name] <String> [-Slot] <String>
+ [-AzureStoragePath <WebAppAzureStoragePath[]>] [-AlwaysOn <Boolean>] [-MinTlsVersion <String>] 
+ [-HealthCheckPath <String>] [-FtpsState <String>] [-ResourceGroupName] <String> [-Name] <String> [-Slot] <String>
```
Comment (panchagnula) [inline src/Websites/Websites/help/Set-AzWebAppSlot.md:28]: Same comment here remove this

File: src/Websites/Websites.Test/ScenarioTests/WebAppSlotTests.ps1 [hunk 3]
```diff
@@ -461,6 +461,7 @@ function Test-SetWebAppSlot
 		$slot.SiteConfig.RequestTracingEnabled = $true
 		$slot.SiteConfig.FtpsState = "FtpsOnly"
 		$slot.SiteConfig.MinTlsVersion = "1.0"
+		$slot.SiteConfig.HealthCheckPath = "/api/path"
```
Comment (panchagnula) [inline src/Websites/Websites.Test/ScenarioTests/WebAppSlotTests.ps1:464]: Update the tests to test using Webapp object  https://docs.microsoft.com/en-us/powershell/module/Az.Websites/set-azwebapp?view=azps-7.3.0

**Review by panchagnula — CHANGES_REQUESTED** [review]
See comments

**Comment by BethanyZhou** [comment]
@Kotasudhakarreddy , @panchagnula , please let me know when this PR is ready

File: src/Websites/Websites/help/Set-AzWebApp.md (hunk 1 above)
Comment (Kotasudhakarreddy) [reply src/Websites/Websites/help/Set-AzWebApp.md]: Yes, updates as per the suggestions.

File: src/Websites/Websites/help/Set-AzWebAppSlot.md (hunk 2 above)
Comment (Kotasudhakarreddy) [reply src/Websites/Websites/help/Set-AzWebAppSlot.md]: Yes, updates as per the suggestions.

File: src/Websites/Websites.Test/ScenarioTests/WebAppSlotTests.ps1 (hunk 3 above)
Comment (Kotasudhakarreddy) [reply src/Websites/Websites.Test/ScenarioTests/WebAppSlotTests.ps1:464]: Yes, I am using Webapp Object here.

**Comment by Kotasudhakarreddy** [comment]
> See comments

Addressed all the comments given.

**Review by BethanyZhou — APPROVED** [review]

**Pull request merged**